utils/local-deployment/limitador-cluster.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: kuadrant-kuadrant-gwapi-gateway-wasm-ratelimits
```

Comment:
and labels? to which Gateway is being applied?

Reply:
We have two gateways so didn't care about narrowing it down to a single gateway.